[2025-09-25T09:45:02.810+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-25T09:45:02.842+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_pipeline_orchestration.check_analytics_api scheduled__2025-09-25T09:30:00+00:00 [queued]>
[2025-09-25T09:45:02.860+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_pipeline_orchestration.check_analytics_api scheduled__2025-09-25T09:30:00+00:00 [queued]>
[2025-09-25T09:45:02.860+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-25T09:45:02.889+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_analytics_api> on 2025-09-25 09:30:00+00:00
[2025-09-25T09:45:02.975+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=1728) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-25T09:45:02.980+0000] {standard_task_runner.py:64} INFO - Started process 1733 to run task
[2025-09-25T09:45:02.894+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_pipeline_orchestration', 'check_analytics_api', 'scheduled__2025-09-25T09:30:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/weather_pipeline_orchestration.py', '--cfg-path', '/tmp/tmpgd0hjd1u']
[2025-09-25T09:45:02.984+0000] {standard_task_runner.py:91} INFO - Job 36: Subtask check_analytics_api
[2025-09-25T09:45:03.457+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_pipeline_orchestration.check_analytics_api scheduled__2025-09-25T09:30:00+00:00 [running]> on host 32ce42b7fa38
[2025-09-25T09:45:03.783+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_pipeline_orchestration' AIRFLOW_CTX_TASK_ID='check_analytics_api' AIRFLOW_CTX_EXECUTION_DATE='2025-09-25T09:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-25T09:30:00+00:00'
[2025-09-25T09:45:03.788+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-25T09:45:03.856+0000] {logging_mixin.py:188} INFO - âœ… Analytics API is healthy
[2025-09-25T09:45:03.856+0000] {python.py:237} INFO - Done. Returned value was: API OK
[2025-09-25T09:45:03.856+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-25T09:45:03.870+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_pipeline_orchestration, task_id=check_analytics_api, run_id=scheduled__2025-09-25T09:30:00+00:00, execution_date=20250925T093000, start_date=20250925T094502, end_date=20250925T094503
[2025-09-25T09:45:03.889+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-25T09:45:03.941+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-25T09:45:03.942+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
